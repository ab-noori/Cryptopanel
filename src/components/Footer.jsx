import React from 'react';
import './Footer.scss';

const Footer = () => (
  <p className="copyright">&copy; 2023 Crypto Panel | Developed by Abdulali Noori</p>
);

export default Footer;
