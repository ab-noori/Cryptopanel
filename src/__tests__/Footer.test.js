import React from 'react';
import { render } from '@testing-library/react';
import Footer from '../components/Footer';

describe('Footer component', () => {
  test('renders correctly', () => {
    const { asFragment } = render(<Footer />);
    expect(asFragment()).toMatchSnapshot();
  });
});
