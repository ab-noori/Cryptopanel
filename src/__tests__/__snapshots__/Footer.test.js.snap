// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Footer component renders correctly 1`] = `
<DocumentFragment>
  <p
    class="copyright"
  >
    Â© 2023 Crypto Panel | Developed by Abdulali Noori
  </p>
</DocumentFragment>
`;
